packages/playground/src/pages/sandbox-settings-project-general-page.tsx

```diff
@@ -67,6 +78,14 @@ function SandboxSettingsProjectGeneralPage() {
       resetProjectSettingsForm(data) // Reset to the current values
       setTimeout(() => setSubmitted(false), 2000)
     }, 2000)
+    onFormSubmit(data)
+  }
+
+  // Watch the verification value
+  const verificationValue = watch('verification')
+
+  const typeCheck = (value: string) => {
```

Comment:
This looks a bit oddly named. `typeCheck` sounds like is checking the type of `value`.

Reply:
fixed: verificationValue -> verificationCheck